docs/sdk/v3/guides/02-quoting.md

```diff
@@ -1,6 +1,113 @@
 ---
 id: quoting
-title: Getting a Quote
----
+title: Quoting
+---     
+
+## Introduction
+
+This guide will cover how to get the current quotes for any token pair on Uniswap.
+It is based on the [Quoting code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/quoting), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+
+In this example we will use `quoteExactInputSingle` to get a quote for the pair **USDC - WETH**.
+
+The guide will **cover**:
+1. Computing the **USDC - WETH** Pool's deployment address
+2. Setting up a reference to the Pool smart contract
+3. Getting Pool metadata from the Pool smart contact
+4. Setting up a reference to the Quoter contract
+5. Getting **quotes** from the Quoter contract
+
+The **inputs** to the example are:
+1. Token in
+2. Token out
+3. Amount in
```

Comment:
This feels like it should come before and be in a word summary rather than taking up so much room here